.gitignore

```diff
@@ -113,3 +113,10 @@ src/.vscode/
 /src/.vs
 /src/CppProperties.json
 /src/launch.vs.json
+
+# Python-related files
+*.pyc
+__pycache__/
+*.pyo
+*.pyd
+**/venv/
```

Comment:
Why two asterisks in `**/venv/`?

Reply:
I think this doesn't really matter. It's redundant in this case but shouldn't harm either. https://stackoverflow.com/questions/41761128/are-leading-asterisks-redundant-in-gitignore-path-matching-syntax

Could be simplified to `venv/` and maybe add `.venv/` as well as it is another common virtualenv name.

Reply:
Two asterisks means that it can be nested any depth. May be redundant, but it definitely works.